src/plugin.ts

```diff
@@ -99,6 +101,14 @@ const plugin: FastifyPluginAsync<AppsPluginOptions> = async (fastify, options) =
       // or using mobile app client (auth token)
       fastify.addHook('preHandler', fastify.verifyAuthentication);
 
+      const aS = new AppService();
+      fastify.decorate('appService', aS);
+
+      fastify.get('/list', { schema: getMany }, async ({ member }) => {
```

Comment:
What's the final endpoint? `/app-items/list`?

Reply:
yes, it's /app-items/list